mlflow/gateway/constants.py

```diff
@@ -0,0 +1,6 @@
+GATEWAY_VERSION = "0.1.0"
+CONF_PATH_ENV_VAR = "MLFLOW_GATEWAY_CONFIG_PATH_ENV_VAR_NAME"
+LOCALHOST = "127.0.0.1"
```

Comment:
This is only used in `tests`; can we move it there?

Reply:
oops. Forgot to remove that. Using the helpers function definition now.